service/src/main/java/uk/gov/hmcts/reform/fpl/service/cmo/DraftOrderService.java

```diff
@@ -197,10 +185,12 @@ public UUID updateCase(UploadDraftOrdersData eventData, List<Element<HearingBook
                 .flatMap(id -> findElement(id, hearings))
                 .orElse(null);
 
-            eventData.getCurrentHearingOrderDrafts().forEach(o -> {
+            for (int i = 0; i < eventData.getCurrentHearingOrderDrafts().size(); i++) {
+                Element<HearingOrder> o = eventData.getCurrentHearingOrderDrafts().get(i);
```

Comment:
It doesn't really matter but might be nice to have "o" as "orderDraft" or something more descriptive

Reply:
👍